java/opendcs/src/main/java/decodes/cwms/validation/ScreeningAlgorithm.java

```diff
@@ -358,4 +352,4 @@ public String[] getPropertyNames()
 	{
 		return _propertyNames;
```

Comment:
Can these methods be removed?
public String[] getInputNames()
public String[] getOutputNames()
public String[] getPropertyNames()